integrations/elasticsearch/documentation.yaml

```diff
@@ -0,0 +1,107 @@
+exporter_type: standalone
+app_name_short: Elasticsearch
+app_name: {{app_name_short}}
+app_site_name: Elasticsearch
+app_site_url: https://www.elastic.co/elasticsearch/
+exporter_name: the Elasticsearch exporter
+exporter_pkg_name: elasticsearch_exporter
+exporter_repo_url: https://github.com/prometheus-community/elasticsearch_exporter
+dashboard_available: true
+minimum_exporter_version: v1.3.0
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+standalone_config: |
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: elasticsearch-exporter
+    labels:
+      app.kubernetes.io/name: elasticsearch-exporter
+  spec:
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: elasticsearch-exporter
+    template:
+      metadata:
+        labels:
+          app.kubernetes.io/name: elasticsearch-exporter
+      spec:
+        containers:
+        - name: exporter
+          image: quay.io/prometheuscommunity/elasticsearch-exporter:v1.3.0
+          args:
+            - '--es.uri=https://user:password@elasticsearch-es-internal-http:9200'
+          ports:
+          - containerPort: 9114
+            name: prometheus
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: elasticsearch
+    labels:
+      app.kubernetes.io/name: elasticsearch
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+    - port: prometheus
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+        app.kubernetes.io/name: elasticsearch-exporter
+additional_prereq_info: |
```

Comment:
This seems like "additional_install_info" (it makes sense after you've seen the install config).